src/styles/card.scss

```
@@ -0,0 +1,78 @@
+.card {
+  padding: 14px 14px 15px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+
+  background-color: #fff;
+}
+
+.card__image {
+  @include card-image(160px,134px,16px);
+  // (width, height, margin-top)
```

```suggestion
  border: 1px solid #f3f3f3;

  background-color: #fff;
  
 &__image {
  @include card-image(160px,134px,16px);
  // (width, height, margin-top)
}
```
 Во всех файлах используй scss вложенность